### DISCUSSION TITLE
Large Gear Shaft issues

### DISCUSSION THREAD

**Discussion by tootechy** [doudar/SmartSpin2k#249] [General]
Anyone have issues w/ printing or issues w/ the large gear shaft? I'm having some difficulty getting the shaft to print properly (probably a cooling issue). But I'm also wondering if the shaft would be stronger as a separate print, laying on the bed, not printing as a column where the shaft could split along the layers.

Anyone else have issues w/ the print or thoughts on the shaft printing w/ the flat side agains the bed?

**Comment by doudar** [comment]
Probably increase your minimum layer time or increase your cooling. 

Typically when I print these, I will print more than one at a time or have another part next to it with equal or higher height. That will help the last layer to firm up before printing the next layer.

**Reply by tootechy** [reply]
Good call, I forgot about that benefit of printing multiple items. I’ll give that a shot. It’s a 10 year old printer, works well, but not the best resolution and some things just don’t print well 😒

Thanks for the reply!

Neil

**Comment by doudar** [comment]
Here's the prusaSlicer profile I use to print everything but the insert. This is for a pretty heavily modified CR-10 and a .6mm nozzle but it might be a good starting point for you if you're having problems. 
[Everything_June.zip](https://github.com/doudar/SmartSpin2k/files/7549178/Everything_June.zip)

**Reply by tootechy** [reply]
Thanks very much for that! Your first suggestion of printing multiple worked perfectly, I printed 3 and they came out fine.

Thanks for the suggestion!

**Comment by tootechy** [comment]
The hex bolt design will be a great improvement. I'm planning on reprinting with that design. Thanks!

**Reply by doudar** [reply]
Awesome! Glad you like it - it's been working great and since it's a drop in replacement, an easy upgrade.
